cesium_app/flask_server.py

```diff
@@ -52,74 +52,80 @@ class UnauthorizedAccess(Exception):
     pass
 
 
+def error(message):
+    return to_json(
+        {
+            "status": "error",
+            "message": message
+        })
+
+def success(data, action=None, payload=None):
```

Comment:
Should `data` also be an optional keyword arg? Oftentimes we return a success message with no data